webapp/ansible/roles/service-vm/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+  - name: Fix Debian locale error.
+    shell: echo 'LANGUAGE="en_US.UTF-8"' > /etc/default/locale && echo 'LC_ALL="en_US.UTF-8"' >> /etc/default/locale # The first redirection intentionally creates the file.
```

Comment:
Maybe add a `creates=...` clause here, or you may use update-locale command, i.e.

```
- command: update-locale LANG=en_US.UTF-8 creates=/etc/default/locale
```